dashboard/app/cache.go

```diff
@@ -208,29 +208,49 @@ func minuteCacheNsUpdate(c context.Context, ns string) error {
 }
 
 func CachedManagerList(c context.Context, ns string) ([]string, error) {
-	key := fmt.Sprintf("%s-managers-list", ns)
+	return cachedObjectList(c,
+		fmt.Sprintf("%s-managers-list", ns),
+		time.Minute,
+		func(c context.Context) ([]string, error) {
+			return managerList(c, ns)
+		},
+	)
+}
+
+func CachedUIManagers(c context.Context, accessLevel AccessLevel, ns string,
+	filter *userBugFilter) ([]*uiManager, error) {
+	return cachedObjectList(c,
+		fmt.Sprintf("%s-%v-%v-ui-managers", ns, accessLevel, filter.Hash()),
```

Comment:
Do we have any tests that ensure we can't screw up access levels with cached objects?

Reply:
I think we could actually get rid of access level here. It was only needed because this function takes it:

https://github.com/google/syzkaller/blob/b5729d823e22b9aada3c12f2bbb218c278df1acf/dashboard/app/main.go#L2140-L2152

Looks like it's only relevant to distinguish user/admin access for the managers list on the admin page, because managers themselves do not have access levels, only namespaces.

Reply:
What about cached bugs?

Reply:
For cached bugs, we do have tests: https://github.com/google/syzkaller/blob/master/dashboard/app/cache_test.go